Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
That's a great explanation, could you add it in a code comment, so that it is not lost? Or maybe make the code more explicit about that? It's hard to understand…